src/features/content-explorer/content-explorer/ContentExplorerHeaderActions.js

```diff
@@ -185,13 +190,22 @@ class ContentExplorerHeaderActions extends Component {
                     )}
                     {children}
                 </div>
-                <ContentExplorerBreadcrumbs
-                    breadcrumbProps={breadcrumbProps}
-                    foldersPath={foldersPath}
-                    isUpButtonDisabled={foldersPath.length <= 1 && !isInSearchMode}
-                    onUpButtonClick={this.handleBreadcrumbsUpButtonClick}
-                    onBreadcrumbClick={this.handleBreadcrumbClick}
-                />
+                {isIncludeSubfoldersAllowed ? (
+                    <ContentExplorerFolderTreeBreadcrumbs
+                        foldersPath={foldersPath}
+                        isFolderTreeButtonDisabled={foldersPath.length <= 1 && !isInSearchMode}
```

Comment:
`foldersPath.length <= 1 && !isInSearchMode` is repeated on line #204, perhaps we should consider storing it in variable.

Reply:
Yeah, good idea